common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4673,6 +4673,41 @@
     },
     "sqlState" : "0A000"
   },
+  "UNSUPPORTED_SHOW_CREATE_TABLE" : {
+    "message" : [
+      "Unsupported show create table command:"
+    ],
+    "subClass" : {
+      "ON_SPARK_DATA_SOURCE_TABLE_WITH_AS_SERDE" : {
```

Comment:
@panbingkun  I changed this error class to a more accurate name and to avoid ambiguity with `WITH_UNSUPPORTED_SERDE_CONFIGURATION`.